### PR TITLE
Added bestvideogame.deals to sites.yml

### PR DESCRIPTION
**Pull request by Koen87**
Added bestvideogame.deals to sites.yml

### CODE REVIEW

**Comment by KyleAMathews** [comment]
Deploy preview for *using-postcss-sass* failed.

Built with commit 6ace6b84956d72dcbfbf4ee6914e4a71df842eb1

https://app.netlify.com/sites/using-postcss-sass/deploys/5b7065303813f0331858f68d

**Comment by KyleAMathews** [comment]
Deploy preview for *gatsbyjs* failed.

Built with commit 6ace6b84956d72dcbfbf4ee6914e4a71df842eb1

https://app.netlify.com/sites/gatsbyjs/deploys/5b70652f3813f0331858f682

File: docs/sites.yml [hunk 1]
```diff
@@ -2078,3 +2078,14 @@
   built_by: Mill3 Studio
   built_by_url: https://mill3.studio/en/
   featured: false
+  - title: Bestvideogame.deals
```
Comment (wKovacs64) [inline docs/sites.yml]: Hey @Koen87, can you remove two spaces from the beginning of the `title` line so the `-` is the first character, please? It's currently invalid YAML. Thanks!

File: docs/sites.yml [hunk 2]
```diff
@@ -2078,3 +2078,14 @@
...
+  main_url: https://bestvideogame.deals/
+  url: https://bestvideogame.deals/
+  description: >
+    Video game comparison website for the UK, build with GatsbyJS.
+  categories:
+    - Price comparison
+    - Video gaming
```
Comment (wKovacs64) [inline docs/sites.yml]: I'd probably change the categories to eCommerce and Video Games, but that's subjective so I'll leave that up to you. See https://github.com/gatsbyjs/gatsby/blob/master/docs/docs/site-showcase-submissions.md#categories for more information on that.

Comment (Koen87) [reply docs/sites.yml]: Changed! Thanks @wKovacs64

File: docs/sites.yml (hunk 1 above)
Comment (Koen87) [reply docs/sites.yml]: Removed!

**Comment by KyleAMathews** [comment]
Deploy preview for *using-contentful* failed.

Built with commit efc8d4f74b9bed5975678be2d43e3aa031bb4c1c

https://app.netlify.com/sites/using-contentful/deploys/5b74039182d3f12b596b427d

**Pull request merged**